jenkins/tests/test_bokchoy_db_pull_request.py

```diff
@@ -16,26 +16,25 @@ class BokchoyPullRequestTestCase(TestCase):
     # Create the Cli runner to run the main function with click arguments
     runner = CliRunner()
 
-    @patch('jenkins.bokchoy_db_pull_request.authenticate_with_github',
+    @patch('jenkins.bokchoy_db_pull_request.github_helper.authenticate_with_github',
            return_value=None)
-    @patch('jenkins.bokchoy_db_pull_request.connect_to_repo',
+    @patch('jenkins.bokchoy_db_pull_request.github_helper.connect_to_repo',
            return_value=None)
-    @patch('jenkins.bokchoy_db_pull_request.get_modified_files_list',
+    @patch('jenkins.bokchoy_db_pull_request.github_helper.get_modified_files_list',
            return_value=None)
-    @patch('jenkins.bokchoy_db_pull_request.branch_exists',
+    @patch('jenkins.bokchoy_db_pull_request.github_helper.branch_exists',
            return_value=False)
-    @patch('jenkins.bokchoy_db_pull_request.create_branch',
+    @patch('jenkins.bokchoy_db_pull_request.github_helper.create_branch',
            return_value=None)
-    @patch('jenkins.bokchoy_db_pull_request.update_list_of_files',
+    @patch('jenkins.bokchoy_db_pull_request.github_helper.update_list_of_files',
            return_value=None)
-    @patch('jenkins.bokchoy_db_pull_request.create_pull_request',
+    @patch('jenkins.bokchoy_db_pull_request.github_helper.create_pull_request',
```

Comment:
I wonder why these were previously mocking the functions as imported into the `jenkins.bokchoy_db_pull_request` module, rather than just mocking them directly in `jenkins.github_helpers`.

Reply:
maybe because we've to mock the methods/functions where they are called instead of where they are defined.